public_html/wp-content/plugins/wordcamp-organizer-survey/includes/debrief-survey/cron.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace WordCamp\OrganizerSurvey\DebriefSurvey\Cron;
+
+defined( 'WPINC' ) || die();
+
+use CampTix_Plugin;
+
+use function WordCamp\OrganizerSurvey\DebriefSurvey\Email\{get_email_id, queue_survey_email};
+use function WordCamp\OrganizerSurvey\DebriefSurvey\Page\{disable_page, publish_survey_page};
+
+/**
+ * Constants.
+ */
+const DAYS_AFTER_TO_SEND = 2;
+const TEMP_ATTENDEE_ID   = 'organizer_debrief_survey_temp_attendee';
+
+/**
+ * Actions & hooks
+ */
+add_action( 'init', __NAMESPACE__ . '\schedule_jobs' );
+add_action( 'wc_organizer_debrief_survey_email', __NAMESPACE__ . '\queue_organizer_survey' );
+add_action( 'wc_organizer_disable_debrief_survey', __NAMESPACE__ . '\disable_organizer_survey' );
+
+/**
+ * Logs a message to the CampTix email log.
+ */
+function log( $message, $post_id, $data = null ) {
+	/* @var CampTix_Plugin $camptix */
+	global $camptix;
+
+	$camptix->log( $message, $post_id, $data );
+}
+
+/**
+ * Get Wordcamp attendee ID.
+ *
+ * @return int
+ */
+function get_wordcamp_attendees_id() {
+	return get_option( TEMP_ATTENDEE_ID );
+}
+
+/**
+ * Add cron jobs to the schedule.
+ */
+function schedule_jobs() {
+	if ( ! wp_next_scheduled( 'wc_organizer_debrief_survey_email' ) ) {
+		wp_schedule_event( time(), 'daily', 'wc_organizer_debrief_survey_email' );
+	}
+}
+
+/**
+ * Add temporary attendee since email can only be sent to items in attendee tracker.
+ */
+function add_temp_attendee() {
+	if ( ! get_wordcamp_attendees_id() ) {
+		$attendees_id = wp_insert_post( array(
+			'post_title' => 'Organizer debrief survey',
+			'post_name' => 'organizer-debrief-survey',
+			'post_type' => 'tix_attendee',
+			'post_status' => 'publish',
+		) );
+
+		if ( $attendees_id ) {
+			$meta = get_wordcamp_post( get_current_blog_id() )->meta;
+			update_post_meta( $attendees_id, 'tix_email', $meta['Email Address'][0] );
+			update_post_meta( $attendees_id, 'tix_receipt_email', $meta['Email Address'][0] );
+			update_post_meta( $attendees_id, 'tix_first_name', $meta['Organizer Name'][0] );
+
+			update_option( TEMP_ATTENDEE_ID, $attendees_id );
+			log( 'Successfully added attendee:', get_email_id(), $attendees_id );
+		} else {
+			log( 'Failed to add attendee:', get_email_id(), $attendees_id );
+		}
+	}
+}
+
+/**
+ * Associates attendee to emails.
+ */
+function associate_attendee_to_email( $email_id ) {
+	$recipient_id = get_wordcamp_attendees_id();
+
+	if ( empty( $recipient_id ) ) {
+		log( 'No valid recipients', $email_id, null );
+		return;
+	}
+
+	// Associate attendee to tix_email as a recipient.
+	$result = add_post_meta( $email_id, 'tix_email_recipient_id', $recipient_id );
+
+	if ( ! $result ) {
+		log( 'Failed to add recipients:', $email_id, $recipient_id );
+	} else {
+		update_post_meta( $email_id, 'tix_email_recipients_backup', (array) $recipient_id );
```

Comment:
Did you notice this happening, or is it more to protect against a theoretical situation?

Reply:
Yes, I noticed there's a warning
```
Warning: count(): Parameter must be an array or an object that implements Countable in /usr/src/public_html/wp-content/plugins/camptix/camptix.php on line 808
```
<img width="1534" alt="image" src="https://github.com/WordPress/wordcamp.org/assets/18050944/9a5b2875-70eb-4a16-92d4-7f7fc0fd79fa">

After looking into the issue, it would get an empty string as recipients_backup at this line
https://github.com/WordPress/wordcamp.org/blob/ea44ddd8008b565b3b32da99be3149aacad12d38/public_html/wp-content/plugins/camptix/camptix.php#L807
This would make it counted as 1. I've pushed this [commit](ea44ddd8008b565b3b32da99be3149aacad12d38) to fix it.